enterprise/releases/2024-04-3.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: '2024.04.3 Release'
+backlink_href: /docs/enterprise/releases
+backlink_title: 'pganalyze Enterprise Server: Release Changelog'
+---
+
+**Docker tag:** `quay.io/pganalyze/enterprise:v2024.04.3`
+
+## Notes
+
+- This is a patch release on top of **[2024.04.0](2024-04-0)**, **[2024.04.1](2024-04-1)** and **[2024.04.1](2024-04-2)**
```

Comment:
Typo: the last link's display text should be 2024.04.2

Reply:
good catch!